unittests/test_policies.py

```diff
@@ -595,12 +595,13 @@ def _read_timestamps(tasks):
     return begin_stamps, end_stamps
 
 
-def test_concurrency_unlimited(async_runner, make_cases, make_exec_ctx):
+def test_concurrency_unlimited(async_runner, make_cases):
     num_checks = 3
-    # make_exec_ctx(options=max_jobs_opts(num_checks))
 
     runner, monitor = async_runner
-    runner.policy._rfm_max_jobs = num_checks
+    runner.policy._max_jobs = {
+        '_rfm_local': num_checks
+    }
```

Comment:
You should still do it with the `make_exec_ctx` except that the `max_jobs_opts()` should also set the `max_local_jobs` option.

Reply:
It doesn't work with the `make_exec_ctx`, because the runtime is changed before the asynchronous execution policy is created and the `max_loca_jobs` is read during the construction of the policy, so it gets the default value of 8.